modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -96,7 +101,14 @@ def create
   # Edit page is very similar to new page, except that we don't need to set
   # default attribute values because the object already exists
   # Called by: Global app/config/routes.rb to serve Web page
-  def edit; end
+  def edit
+    # Replace the client secret by **** for security reasons.
+    # A minimum length validation makes sure the old secret isn't
+    # easily overwritten.
+    if @object.oauth_client_secret.present?
+      @object.oauth_client_secret = "****"
```

Comment:
@fraber If you do that, then the next time you save the storage you overwrite the oauth_client_secret with `****`.